src/components/Catalog.vue

```diff
@@ -255,6 +263,13 @@ export default {
       externalItemCount: 0,
       externalItemsPerPage: 0,
       externalItemPaging: false,
+      externalPaginationRelations: ['first', 'previous', 'next', 'last'],
```

Comment:
```suggestion
      externalPaginationRelations: ['first', 'prev', 'next', 'last'],
```

The relation type for previous links is `prev`. IANA also specifies `previous` in addition so we may want to allow both if possible.

Reply:
Ok, I'll find a way to have both

Reply:
so in the end I went for `['first', 'prev', 'previous', 'next', 'last'],` here, add also a `prev` attribute to the `externalPaginationLinks` data, and sort this out in the template.
I don't have a backend with a `prev` link, do you?

Reply:
Solution is fine for now. I'll need to check who actually follows the spec ;-) All "previous" back-ends are actually not really spec compliant.